servers/graphql-kotlin-spring-server/src/test/kotlin/com/expediagroup/graphql/server/spring/SchemaConfigurationTest.kt

```diff
@@ -173,6 +180,9 @@ class SchemaConfigurationTest {
 
         @Bean
         fun myDataLoaderRegistryFactory(): KotlinDataLoaderRegistryFactory = mockk()
+
+        @Bean
+        fun myCustomSchemaPrinter(): SchemaPrinter = mockk(relaxed = true)
```

Comment:
This requires a relaxed mock because the `SdlRouteConfiguration` calls this class in its constructor, so a strict mock throws a bean instantiation error before the relevant tests here can even begin.